### DISCUSSION TITLE
add Line login at third party login

### DISCUSSION THREAD

**Discussion by lv-kit** [supabase/supabase#20178] [Feature Requests] [answered]
Hello everyone.

This text is translated using DeepL and I hope you will read it considering that you may not have used the proper words or sentences.

When providing a login method for the younger generation in Japan, I believe that being able to use LINE Login will lower the hurdle considerably. This is because the younger generation in Japan usually uses LINE messages rather than email applications for communication, so they are not logged out on a daily basis.
"Of LINE's 95 million monthly users, 86% (as of the end of June 2023) use LINE at least once a day, a high usage rate. The highest frequency of use is among males in the 20-24 age group and females in the 25-29 age group."
[https://campus.line.biz/line-ads/courses/user/lessons/oada-1-2-2](https://campus.line.biz/line-ads/courses/user/lessons/oada-1-2-2)

This is why I am clamoring for line login to be available on supabase auth.

Thanks.

**Comment by dninomiya** [comment]
LINE login is an extremely popular login method in Japan. Please consider it as it would significantly contribute to the adoption of Supabase in Japan.

**Reply by jerry-rlnk** [reply]
Same here at Taiwan!

**Reply by sunncyn** [reply]
Same here at Thailand.

**Comment by poteboy** [comment]
@dshukertjr 
Hi, I am desperately longing for this feature. Could you escalate this to the team to add it to the roadmap? I believe you can convince them how important LINE login is to Japanese users.

**Comment by sunncyn** [comment]
been trying to hack to integrate with line login for 2 days (LIFF). not found proper solution. So I moved whole supabase-related snippet to server-side (but I believe supabase sdk was designed with client-side in mind...)

**Comment by dshukertjr** [comment, marked as answer]
There is a separate discussion for consolidating what providers we support here: https://github.com/supabase/auth/issues/451#issuecomment-2229907351

If you want Supabase to add support for something, go ahead and vote over there. I'm going to close this one so that people will actually go to the correct one.

**Comment by The-Poe** [comment]
Need Line login feature here. (also voted at auth#451
